src/userdev/java/net/minecraftforge/gradle/userdev/DependencyManagementExtension.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * ForgeGradle
+ * Copyright (C) 2018 Forge Development LLC
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301
+ * USA
+ */
+
+package net.minecraftforge.gradle.userdev;
+
+import groovy.lang.GroovyObjectSupport;
+import net.minecraftforge.gradle.userdev.util.DependencyRemapper;
+import org.gradle.api.Project;
+import org.gradle.api.artifacts.Dependency;
+
+public class DependencyManagementExtension extends GroovyObjectSupport {
+    public static final String EXTENSION_NAME = "fg";
```

Comment:
Putting `deobf(...)` on top level would make it ambiguous with the deobf configuration. Similarly, the minecraft extension is also ambiguous with minecraft configuration. For now, made it `fg.deobf(...)`. Any other suggestions?